### PR TITLE
Use version named like rails

### PR DESCRIPTION
**Pull request by christophe-dufour**
change the file name from `0001_XXX.up.cql` to `20150811214556_XXX.up.cql`

### CODE REVIEW

File: migrate/migrate.go [hunk 1]
```diff
@@ -17,6 +18,8 @@ import (
 	pipep "github.com/heetch/migrate/pipe"
 )
 
+var nowFunc = time.Now
```
Comment (skateinmars) [inline migrate/migrate.go]: You could have this function return a timestamp instead of having to format it every time?

**Comment by skateinmars** [comment]
:eyes: done

File: migrate/migrate.go (hunk 1 above)
Comment (christophe-dufour) [inline migrate/migrate.go]: I'm not sure I understand your point?

For me the `nowFunc` is only here to be able stub the time, like `timecop` in rails http://labs.yulrizka.com/en/2014/10/stubbing-time-dot-now-in-golang.html#.VcsJxxOqpBc

Comment (skateinmars) [inline migrate/migrate.go]: You only need the current time to generate a timestamp, so if your function directly returns a timestamp directly you can simplify both your application and test code

**Comment by christophe-dufour** [comment]
:up:

**Comment by sush** [comment]
:+1:

**Comment by skateinmars** [comment]
:+1:

**Pull request merged**